### ISSUE TITLE
Question about cancelation for shipment

### ISSUE THREAD

**Issue by yangcen6** [allegro/allegro-api#9614]
If we use ship with allegro,：
1. Can we split the order into multiple packages though API?
2. When the Allegro courier comes to our warehouse to pick up the package, do we need to actively change the status of our order to SENT though **/order/checkout-forms/{id}/fulfillment**? Or will it be automatically changed after the courier is picked up?
3. Under what conditions can we not apply to cancel the shipments though **/shipment-management/shipments/cancel-commands**? For example, under what order status?

**Comment by Lukasz-Zurek** [comment]
1. The maximum number of packages included in one shipment is: 10, but applies only to DPD and WE|DO. Another thing - the maximum number of packages for each carrier is: 10. So you can create many packages for one order. These will be separate packages.
2. Using of this resource is up to you. It will not be automatically changed by Allegro.
3. There are no dependencies with orders here. Please read our [tutorial](https://developer.allegro.pl/tutorials/how-to-manage-parcels-via-ship-with-allegro-ZM9YAyGKWTV#how-to-cancel-a-shipment). There are two situations where you cannot cancel a shipment.

**Comment by yangcen6** [comment]
1. So only these two carriers can split an order into multiple packages? Which endpoint do we need to use to split the packages?
2. That is to say, even if we use ship with allegro, we must change the status of the order through this endpoint to inform the customer that the order has been shipped out or has been delivered?
3. Can I cancel the shipment after it's shipped?

**Comment by Lukasz-Zurek** [comment]
1. You can split the packages, not order. You can create multiple packages using the resource for creating shipments [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment). Please look at the sample request and "packages" section in this [guide](https://developer.allegro.pl/tutorials/how-to-manage-parcels-via-ship-with-allegro-ZM9YAyGKWTV#how-to-create-a-new-shipment).
2. Yes. But please read this [tutorial](https://developer.allegro.pl/tutorials/process-orders-PgPMlWDr8Cv#order-fulfillment-status-change). If the seller has [subscription](https://allegro.pl/moje-allegro/sprzedaz/abonamenty), he can manage some automatic actions [here](https://allegro.pl/moje-allegro/sprzedaz/zamowienia/ustawienia).
3. You cannot cancel the shipments only in two situations which we have mentioned [here](https://developer.allegro.pl/tutorials/how-to-manage-parcels-via-ship-with-allegro-ZM9YAyGKWTV#how-to-cancel-a-shipment).

**Comment by yangcen6** [comment]
For example, there are two items A and B in an order, and we want them to be shipped in different packages. We need to call this endpoint [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment) and pass the relevant information of the two packages in the field **package**，right？

**Comment by Lukasz-Zurek** [comment]
If you ship with DPD or WE|DO - it will be possible to provide more than one package in the **"packages"** section. Other than that - you will have to call [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment) twice for A and B respectively.

**Comment by yangcen6** [comment]
OK，got it, thanks!

**Comment by yangcen6** [comment]
If we use ship with allegro and choose dpd or we|do delivery, if we want to separate A and B in an order into two packages, can we directly create two shipments with [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment) ?

**Comment by Lukasz-Zurek** [comment]
@yangcen6 Yes, in this situation - if you want to have two different waybills = you should create two shipments using [POST /shipment-management/shipments/create-commands](https://developer.allegro.pl/documentation/#operation/createNewShipment).
